util/azure/utils.go

```diff
@@ -359,9 +416,12 @@ func fetchDataActionsList(settings *DiscoverResourcesSettings) ([]Operation, err
 	}
 
 	if resp.StatusCode != http.StatusOK {
+		counterGetOperationsResources.WithLabelValues(ConvertIntToString(resp.StatusCode)).Inc()
```

Comment:
as discussed , cannot move this

Reply:
also i removed this metric as we will never receive this metric in error case. Since, the endpoints of the guard service are assigned once the readiness probe has succeeded. So if it's not ready, prom will not be scrape the metrics anyway